crates/biome_js_analyze/tests/specs/nursery/noUselessTernary/invalid.js

```diff
@@ -1,12 +1,14 @@
-var a = x ? true : true;
-var a = x ? false : false;
+var a = x? true:true;
+var a = x? false:false;
```

Comment:
I know you wanted to test trivia inside the code action, but changing the existing tests will make the review harder.

It's better to add new ones